src/Error.html

```diff
@@ -14,14 +13,6 @@
 <div style="align-items: center; margin: auto; color: red; width:40%;">
 <h1>404: An Error Has Occurred.</h1>
 </div>
-
-
-
-
-
-
-
-
```

Comment:
The empty lines in the code could be removed to have overall cleaner code. This can be applied to multiple files. I feel this would be more in line with Baeldung's source file structure section under clean code standards. https://www.baeldung.com/java-clean-code#3-source-file-structure